website/docs/tutorials/07-testing/02-cheatcodes/skip.md

```diff
@@ -0,0 +1,31 @@
+# `skip`
+```python
+def skip(reason: Optional[str] = None) -> None:
+```
+Skip a test.
```

Comment:
I would like to have info here why is this useful.

Reply:
Done